packages/snap-preact/src/Instantiators/RecommendationInstantiator.tsx

```diff
@@ -127,6 +128,9 @@ export class RecommendationInstantiator {
 				if (options?.categories) {
 					globals.categories = options.categories;
 				}
+				if (options?.limit && Number.isInteger(Number(options.limit))) {
+					globals.limit = Number(options.limit);
```

Comment:
I don't think so because multiple recommendations on the page share the same config here. I tried setting `this.config.config.limits = Number(options.limit);` here and if you have the following:
```
<script type="searchspring/recommend" profile="similar">
      product = "C-AD-W1-1869P";
  </script>
  <script type="searchspring/recommend" profile="similar">
      product = "C-AD-W1-1869P";
      options = {
          limit: 4
      };
  </script>
  <script type="searchspring/recommend" profile="similar">
      product = "C-AD-W1-1869P";
  </script>
  ```
Then you'll see limits: 20, 4, 4. However it works as expected when setting `globals.limits` instead (20, 4, 20)

Then if you were to set `instantiators.recommendations.config.limits = 2` you can change the default (2, 4, 2) and if you set `instantiators.recommendations.config.globals.limits = 3` you can overwrite everything (3, 3, 3)

Reply:
I mean setting the config on the controller being instantiated, not on the instantiator.